src/styles/utils/__normalize.scss

```diff
@@ -0,0 +1,9 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-size: 12px;
+  font-family: Roboto, sans-serif;
```

Comment:
Checklist:
2. [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_product-cards/assets/63243136/f9f701a6-11d2-41c2-a2bd-3f799190c511)